scrapy/crawler.py

```diff
@@ -173,7 +173,7 @@ def stop(self):
 
         Returns a deferred that is fired when they all have ended.
         """
-        return defer.DeferredList([c.stop() for c in list(self.crawlers)])
+        return defer.DeferredList(c.stop() for c in list(self.crawlers))
```

Comment:
Ok. Thank you.

About crawlers. I can only suppose that it could be non iterable object...